lib/tool_belt/outdated_deprecation_warning_finder.rb

```diff
@@ -0,0 +1,61 @@
+module ToolBelt
+  class OutdatedDeprecationWarningFinder
+    attr_accessor :deprecations
+    attr_accessor :current_version
```

Comment:
I see instance methods for current_version and next_minor_version so these may not be necessary